### ISSUE TITLE
Numpy Error- What'd i do wrong?

### ISSUE THREAD

**Issue by kieranxsomer** [bmaltais/kohya_ss#1379]
Oh hell LOL - it's me again: 

2023-08-13 08:14:11.912022: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2023-08-13 08:14:12.582965: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
found 0 images.
loading existing metadata: dataset/Osea_XL/images_new/meta_cap.json
load VAE: dataset/Osea_XL-V1-Alpha.safetensors
0it [00:00, ?it/s]
/workspace/kohya_ss/venv/lib/python3.10/site-packages/numpy/core/fromnumeric.py:3432: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/workspace/kohya_ss/venv/lib/python3.10/site-packages/numpy/core/_methods.py:190: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
mean ar error: nan
writing metadata: dataset/Osea_XL/images_new/meta_lat.json
done!
08:17:55-165679 INFO     Start Finetuning...                                    
08:17:55-167111 INFO     Headless mode, skipping verification if model already  
                         exist... if model already exist it will be             
                         overwritten...                                         
08:17:55-168355 INFO     python3 finetune/prepare_buckets_latents.py            
                         "dataset/Osea_XL/img/"                                 
                         "dataset/Osea_XL/images_new/meta_cap.json"             
                         "dataset/Osea_XL/images_new/meta_lat.json"             
                         "dataset/Osea_XL-V1-Alpha.safetensors" --batch_size=1  
                         --max_resolution=1024,1024 --min_bucket_reso=512       
                         --max_bucket_reso=1024 --mixed_precision=fp16          
08:17:55-170722 INFO     image_num = 0                                          
08:17:55-171556 INFO     repeats = 0                                            
08:17:55-172307 INFO     max_train_steps = 0                                    
08:17:55-173051 INFO     lr_warmup_steps = 0                                    
08:17:55-173871 INFO     Saving training config to                              
                         dataset/Osea_XL/model/osea_xl_alpha-v1b_20230813-081755
                         .json...                                               
08:17:55-175217 INFO     accelerate launch --num_cpu_threads_per_process=2      
                         "./sdxl_train.py"                                      
                         --pretrained_model_name_or_path="dataset/Osea_XL-V1-Alp
                         ha.safetensors"                                        
                         --in_json="dataset/Osea_XL/images_new/meta_cap.json"   
                         --train_data_dir="dataset/Osea_XL/img/"                
                         --output_dir="dataset/Osea_XL/model"                   
                         --logging_dir="dataset/Osea_XL/logs"                   
                         --dataset_repeats=1 --learning_rate=1.5e-06            
                         --enable_bucket --resolution="1024,1024"               
                         --min_bucket_reso=512 --max_bucket_reso=1024           
                         --save_model_as=safetensors                            
                         --output_name="osea_xl_alpha-v1b" --no_half_vae        
                         --learning_rate="1.5e-06"                              
                         --lr_scheduler="constant_with_warmup"                  
                         --train_batch_size="1" --save_every_n_epochs="1"       
                         --mixed_precision="fp16" --save_precision="fp16"       
                         --caption_extension=".txt" --cache_latents             
                         --cache_latents_to_disk --optimizer_type="Adafactor"   
                         --optimizer_args scale_parameter=False                 
                         relative_step=False warmup_init=False                  
                         --max_data_loader_n_workers="0"                        
                         --caption_dropout_rate="0.1" --bucket_reso_steps=64    
                         --min_snr_gamma=5 --mem_eff_attn                       
                         --gradient_checkpointing --xformers --bucket_no_upscale
                         --noise_offset=0.05 --adaptive_noise_scale=0.005       
08:17:55-179193 INFO     The command is already running. Please wait for it to  
                         finish.                                                
2023-08-13 08:17:56.896066: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2023-08-13 08:17:57.557987: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
found 0 images.
loading existing metadata: dataset/Osea_XL/images_new/meta_cap.json
load VAE: dataset/Osea_XL-V1-Alpha.safetensors
0it [00:00, ?it/s]
/workspace/kohya_ss/venv/lib/python3.10/site-packages/numpy/core/fromnumeric.py:3432: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/workspace/kohya_ss/venv/lib/python3.10/site-packages/numpy/core/_methods.py:190: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
mean ar error: nan
writing metadata: dataset/Osea_XL/images_new/meta_lat.json
done!

This MIGHT be related to one of the pull requests but i'm sure i'm certain i've done a dumb.
Unlike Rocker's one, where it IS finding images, this one isn't and it's giving me a numpy error lol. 

I've got like 1200 images in that folder lol. 
Maybe i need to structure it better i'm not sure but i'm not in a huge rush 

I'm running on Runpod - have had 0 issues doing lora, i'm on an RTX 3090 ti. 
I'm still blaming me, not the code.

**Comment by kieranxsomer** [comment]
Ok I FIXED that using a terminal workaround: python  ./finetune/merge_captions_to_metadata.py dataset/Osea_XL/images_new 'dataset/Osea_XL/images_new/meta-cap.json' --caption_extension .txt
